vector/src/main/java/im/vector/activity/VectorRoomInviteMembersActivity.java

```diff
@@ -244,20 +277,21 @@ public boolean onChildClick(ExpandableListView parent, View v, int groupPosition
                 final Object item = mAdapter.getChild(groupPosition, childPosition);
 
                 if (item instanceof ParticipantAdapterItem) {
+                    final ParticipantAdapterItem participantItem = (ParticipantAdapterItem) mAdapter.getChild(groupPosition, childPosition);
                     if (null != mMode && mMode == VectorRoomCreationActivity.RoomCreationModes.DIRECT_CHAT) {
-
-                        final ParticipantAdapterItem participant = (ParticipantAdapterItem) mAdapter.getChild(groupPosition, childPosition);
-                        DinsicUtils.startDirectChat(VectorRoomInviteMembersActivity.this, mSession, participant);
-
+                        DinsicUtils.startDirectChat(VectorRoomInviteMembersActivity.this, mSession, participantItem);
                     } else {
-                        addParticipantToListToInvite((ParticipantAdapterItem) item);
+                        updateParticipantListToInvite(participantItem);
+                        mAdapter.mCurrentSelectedUsers = userIdsToInvite;
+                        mAdapter.notifyDataSetChanged();
+                        invalidateOptionsMenu();
```

Comment:
Can we use invalidateMenu here (see https://github.com/vector-im/riot-android/pull/2339/files#diff-c08463d652245e24cf0d3666b51e048aR130)

Reply:
Not at the moment. This action can be done after the merge of this PR on develop.